server/routes/index.js

```diff
@@ -37,6 +39,17 @@ router.get("/schema/:schemaId", async (req, res) => {
     }
 });
 
+router.post("/testHook", cors(), async (req, res) => {
```

Comment:
It's not a requirement but the other test button that we have (Postgres data source) uses graphql instead of a post request. We could do the same here.